code/game/gamemodes/changeling/powers/adrenaline.dm

```diff
@@ -10,12 +10,10 @@
 //Recover from stuns.
 /obj/effect/proc_holder/changeling/adrenaline/sting_action(mob/living/user)
 	user << "<span class='notice'>Energy rushes through us.[user.lying ? " We arise." : ""]</span>"
-	user.stat = 0
+	user.SetSleeping(0)
 	user.SetParalysis(0)
 	user.SetStunned(0)
 	user.SetWeakened(0)
-	user.lying = 0
-	user.update_canmove()
```

Comment:
these are already called by SetWeakened(), and SetStunned(), and SetParalysis().